test/1.4/structure/T_subobjs.dml

```diff
@@ -37,7 +36,11 @@ bank b[i < 2] {
     saved int bs;
 }
 
-method test() throws {
```

Comment:
If this pattern is common (say, more than twice) we should keep the testing lib and just not use it unless needed

Reply:
It's not that common. Also I don't fully agree -- for simple logic, the value of being explicit and standalone often outweighs the cost of duplication. The attribute is the communication channel between X.py and X.dml, and that channel is easier to see if the attribute is explicitly declared rather than inherited from some lib.